tests/tensorflow/test_printing.py

```diff
@@ -104,13 +104,14 @@ def test_shape_index():
     ... |  Tensor(Add):0,\tshape=[None, 1]\t"x_p_y:0"
     ... |  |  Op(Add)\t"x_p_y"
     ... |  |  |  Tensor(Mul):0,\tshape=[None, 1]\t"y:0"
+    ... |  |  |  ...
     ... |  |  |  Tensor(Mul):0,\tshape=[None, 1]\t"y:0"
-    ... |  |  |  |  ...
+    ... |  |  |  ...
```

Comment:
The ellipses indicate that everything above/below the level they're on is truncated.  This output says (twice) that the level those `Tensor(Mul)`s are on is being truncated.  
In other words, it looks like you need to (re)add a pipe before those ellipses, no?

Reply:
Added the pipe back so the levels should make more sense now.